doc/manual/src/projects.md

```diff
@@ -230,11 +232,15 @@ yield an error like this:
 The error is self-explanatory. Assuming `$HOME/src/hello` points to a
 checkout of Hello, this can be fixed this way:
 
-    $ nix-build -I ~/src release.nix -A tarball
+```console
+$ nix-build -I ~/src release.nix -A tarball
+```
 
 Similarly, the `build` jobset can be evaluated:
 
-    $ nix-build -I ~/src release.nix -A build
+```console
```

Comment:
Not sure if there is any difference between ShellSession and console.

Reply:
mdbook doesn't support ShellSession, but does support console, and GitHub treats console as an alias to ShellSession. :+1: